discovery-provider/plugins/pedalboard/apps/sla-auditor/src/libs.ts

```diff
@@ -0,0 +1,26 @@
+import { AudiusLibs } from "@audius/sdk";
+import HDWalletProvider from '@truffle/hdwallet-provider'
+import Web3 from 'web3'
+
+// TODO: promote this into a packages
+export const initAudiusLibs = async (): Promise<AudiusLibs> => {
```

Comment:
Would you agree it would be worth it to move a libs initializer into base at some point?

Reply:
Yes 100%. This initialization is slightly different from the other one because it needs an "unlocked" wallet to be able to send a main-net eth tx... but the super-set being a package is probably a good idea? Not strongly opinionated though. "wet" isn't horrible for this